### PR TITLE
update cmakelist and readme for unit test

### PR DESCRIPTION
**Pull request by kungyao**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,8 +8,8 @@ option(Boost_USE_MULTITHREAD "Use multithread" ON)
 option(Boost_USE_STATIC_RUNTIME "Use static runtime" ON)
 option(Boost_USE_DEBUG_RUNTIME "Use debug runtime" ON)
 
-# find boost library     date_time 
-find_package(Boost REQUIRED COMPONENTS system thread regex)
+# find boost library # date_time COMPONENTS system thread regex
```
Comment (john0312) [inline CMakeLists.txt:11]: Remove unused comments?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+```
+sudo apt-get update 
+sudo apt install g++
+sudo apt-get install -y libssl-dev
+
+# compile gtest
+# https://zwarrior.medium.com/install-google-test-framework-gtest-on-ubuntu-20-04-368eb6951b12
+sudo apt-get install -y libgtest-dev
+cd /usr/src/gtest
+sudo cmake CMakeLists.txt
```
Comment (john0312) [inline README.md:14]: [Optional] It would preferred to use the system's gtest.

**Review by john0312 — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (kungyao) [reply CMakeLists.txt]: done

File: README.md (hunk 2 above)
Comment (kungyao) [reply README.md:14]: Prefer keeping it just in case.

**Pull request merged**